Refresh.GameServer/Services/CommandService.cs

```diff
@@ -100,6 +100,11 @@ public void HandleCommand(CommandInvocation command, GameDatabaseContext databas
                 
                 break;
             }
+            case "togglegriefphotos": {
+                user.RedirectGriefReportsToPhotos = !user.RedirectGriefReportsToPhotos;
+                
+                break;
+            }
```

Comment:
Would prolly work better as an on/off instead of a toggle. That way it's explicit in what you're doing so you can't get since you can't know the result of the command.

It would be a different story if LBP actually used the result of `/filter` on the client, though. I assume they did that so it's harder to know if you hit the moderation filter.